patches/server/0014-Timings-v2.patch

```diff
@@ -956,22 +956,28 @@ index 59eed41e1c6012650597f1faa85063a2d536873f..51eac8b7177db66c005e4eaca689cf96
      }
  
      @Override
-@@ -667,6 +668,7 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
+@@ -671,7 +672,9 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
+     }
  
-     @Override
-     public String runCommand(String command) {
-+        Waitable[] waitableArray = new Waitable[1];
-         this.rconConsoleSource.prepareForCommand();
+     public String runCommand(RconConsoleSource rconConsoleSource, String s) {
++        Waitable[] waitableArray = new Waitable[1]; // Paper
+         rconConsoleSource.prepareForCommand();
++        final java.util.concurrent.atomic.AtomicReference<String> command = new java.util.concurrent.atomic.AtomicReference<>(s); // Paper
```

Comment:
Only change from noah's PR is a safety net init of the command atomic ref.
If we somehow explode in the executeBlocking, the excption messages have something to fall back onto.